### PR TITLE
fix(create): add podLabels to chart create

### PR DESCRIPTION
**Pull request by genofire**

### CODE REVIEW

**Review by joejulian — APPROVED** [review]
lgtm

File: pkg/chartutil/create.go [hunk 1]
```diff
@@ -308,6 +309,9 @@ spec:
       {{- end }}
       labels:
         {{- include "<CHARTNAME>.selectorLabels" . | nindent 8 }}
+	{{- with .Values.podLabels }}
```
Comment (l2dy) [inline pkg/chartutil/create.go:312]: Line 312 is tab-indented, while others are space-indented. Should they be consistent?

Comment (genofire) [reply pkg/chartutil/create.go:312]: oh sorry

Comment (genofire) [reply pkg/chartutil/create.go:312]: https://github.com/helm/helm/pull/12457

**Pull request merged**